addons/magazinerepack/functions/fnc_canRepackMagazine.sqf

```diff
@@ -19,11 +19,13 @@
 params ["_unit", "_magazine"];
 
 private _maxAmmoCount = getNumber (configFile >> "CfgMagazines" >> _magazine >> "count");
+private _isRepackDisabled = getNumber (configFile >> "CfgMagazines" >> _magazine >> "ace_disableRepacking") == 1;
```

Comment:
Can check this before the count and just an exitWith, avoid two config lookups when disabled

incredibly minor impact though